python/ml4ir/build/Dockerfile

```diff
@@ -1,18 +1,12 @@
 # Using python based centos7 docker base image:
-FROM centos/python-36-centos7:latest
+FROM ducouloa/ml4ir:0.1
```

Comment:
Since is mage is used as part of out ci: https://github.com/salesforce/ml4ir/blob/master/.circleci/config.yml#L7
Built from this Dockerfile https://github.com/salesforce/ml4ir/blob/master/.circleci/images/Dockerfile

Reply:
@Ullimague Should we create an account for salesforce search, perhaps?

Reply:
we could, sure, but not sure if it's needed for now.